scp.1

```diff
@@ -38,9 +37,21 @@ It uses
 .Xr ssh 1
 for data transfer, and uses the same authentication and provides the
 same security as a login session.
-The scp protocol requires execution of the remote user's shell to perform
+Since OpenSSH 8.7,
+.Nm
+has used the SFTP protocol for file transfers and requires a working
+.Xr sftp-server 8
+(or equivalent) to be present on the server.
+Use of the legacy SCP protocol remains available via the
+.Fl O
+option.
+.Pp
+The SCP protocol requires execution of the remote user's shell to perform
+.Xr glob 3
+pattern matching, whereas the SFTP protocol does not.
+This may result in some complex
 .Xr glob 3
-pattern matching.
+patterns being processed differently, depending on the protocol in use.
```

Comment:
The other difference, according to the documentation, is the symlinks processing: SFTP doesn't follow them. SCP does...

Reply:
That is fixed in the last commit in this PR

Reply:
also, this is referring to the protocol and not the sftp(1) implementation